packages/webapp/pages/onboarding.tsx

```diff
@@ -101,9 +110,21 @@ export function OnboardPage(): ReactElement {
   const onSuccessfulTransaction = () => {
     onShouldUpdateFilters(true);
     setFinishedOnboarding(true);
+    if (hasSelectTopics) {
+      return;
+    }
+
     router.push('/');
   };
 
+  useEffect(() => {
+    if (!hasSelectTopics || !alerts?.myFeed) return;
+
+    if (alerts.myFeed === 'created') {
+      router.push('/');
+    }
+  }, [alerts, hasSelectTopics, router]);
+
```

Comment:
Could we not even move this to `hasSelectedTopics` callback and just redirect from there? Sorry for multiple changes, but just realised it now when you adjusted.

Reply:
The issue i found it wasn't ready at that point the alerts took a while to update to created / till it was ready. Doing it in hasSelectedTopics would be too early

Reply:
Ah, of course because of update query.